Linux CLOCK_MONOTONIC is unaffected by admin jumps in time but is
affected by NTP skewing etc so multiple calls may return the same value.
CLOCK_MONOTONIC_RAW is not affected by skewing. Depending on which
behaviour upstream wants, this change may or may not be correct.

Index: winpr/libwinpr/sysinfo/sysinfo.c
--- winpr/libwinpr/sysinfo/sysinfo.c.orig
+++ winpr/libwinpr/sysinfo/sysinfo.c
@@ -328,7 +328,11 @@ BOOL GetSystemTimeAdjustment(WINPR_ATTR_UNUSED PDWORD 
 }
 
 #ifndef CLOCK_MONOTONIC_RAW
-#define CLOCK_MONOTONIC_RAW 4
+#  if defined(__linux__)
+#    define CLOCK_MONOTONIC_RAW 4
+#  else
+#    define CLOCK_MONOTONIC_RAW CLOCK_MONOTONIC
+#  endif
 #endif
 
 DWORD GetTickCount(void)
